microcosm_daemon/healthcheck_server.py

```diff
@@ -34,6 +35,9 @@ def healthcheck():
             else 500
         )
 
+        if status != 200:
+            logger.warning("Healthcheck heartbeat status: UNHEALTHY")
+        logger.debug("Healthcheck heartbeat status: HEALTHY")
```

Comment:
I mention "heartbeat" in the message to differentiate from this other health-related message https://github.com/globality-corp/microcosm-daemon/pull/24/files#diff-20a1ba4e89ff5e8020250f9f3655da7e5b7e5f2f2b78b51480293366de5951d4R38
This latter one will be marked as erroring every time there's an error processing a message, which is different from what we're looking at here.

Reply:
When logging at debug level it's possible to log both "HEALTHY" and "UNHEALTHY" events?